packages/lib/src/core/utils.ts

```diff
@@ -11,3 +11,16 @@ export function processGlobalOptions(globalOptions) {
         return r;
     }, {});
 }
+
+export function getDefaultPropsByCountryCode(countryCode: string): Record<string, any> {
```

Comment:
I think we should distinguish if it is Drop-in or not in here. Otherwise, all payment methods from Finland will get these default props, although it must be currently used only in Drop-in.

It is not a big deal for now, but my point is that I'd rather just pass props if they are supposed to be passed and used by the Component. We can avoid pass props that won't be used.

Sending a flag `isDropin` and then perform the switch based on that would work I guess:

```js
export function getDefaultPropsByCountryCode(countryCode: string, isDropin: boolean): Record<string, any> {
  switch (countryCode) {
    case 'FI': {
       return isDropin ? { openFirstPaymentMethod: false, openFirstStoredPaymentMethod: false } : {}
    }
    default: 
      return {};
  }
}
...
```

Reply:
For this particular rule it makes sense, yes